DQMOffline/Configuration/python/autoDQM.py

```diff
@@ -96,13 +96,13 @@
                                  'PostDQMOffline',
                                  '@common+@muon+@hcal+@jetmet+@ecal+@egamma+@lumi+@L1TMuon+@L1TEgamma+@ctpps'],
 
-            'rerecoZeroBias' : ['@commonSiStripZeroBias+@muon+@hcal+@jetmet+@ecal+@egamma+@L1TMuon+@L1TEgamma+@ctpps',
+            'rerecoZeroBias' : ['DQMOfflineCommonSiStripZeroBias+@muon+@hcal+@hcal2+@jetmet+@ecal+@egamma+@L1TMuon+@L1TEgamma+@ctpps',
                                 'PostDQMOffline',
-                                '@commonSiStripZeroBias+@muon+@hcal+@jetmet+@ecal+@egamma+@L1TMuon+@L1TEgamma+@ctpps'],
+                                'DQMHarvestCommonSiStripZeroBias+DQMCertCommon+@muon+@hcal+@hcal2+@jetmet+@ecal+@egamma+@L1TMuon+@L1TEgamma+@ctpps'],
 
-            'rerecoZeroBiasFakeHLT' : ['@commonSiStripZeroBiasFakeHLT+@muon+@hcal+@jetmet+@ecal+@egamma+@L1TMuon+@L1TEgamma',
-                                'PostDQMOffline',
-                                '@commonSiStripZeroBiasFakeHLT+@muon+@hcal+@jetmet+@ecal+@egamma+@L1TMuon+@L1TEgamma'],
+            'rerecoZeroBiasFakeHLT' : ['DQMOfflineCommonSiStripZeroBiasFakeHLT+@muon+@hcal+@hcal2+@jetmet+@ecal+@egamma+@L1TMuon+@L1TEgamma',
+                                       'PostDQMOffline',
+                                       'DQMHarvestCommonSiStripZeroBiasFakeHLT+DQMCertCommonFakeHLT+@muon+@hcal+@hcal2+@jetmet+@ecal+@egamma+@L1TMuon+@L1TEgamma'],
```

Comment:
@fabferro @vavati @chayanit @fioriNTU 
I think that `@ctppts` tag is missing from the `rerecoZeroBiasFakeHLT` sequence, which is what is actually run in the RelVals as opposed to `rerecoZeroBias` which is what is used for data ReReco and has already been taken care of by this commit https://github.com/cms-sw/cmssw/commit/835a4affdca2c16630807949e9c4c3d9e969c5a2#diff-cbd7e70fcf453b18b0264833a02bcee8.
This is likely the cause of the failure reported in https://hypernews.cern.ch/HyperNews/CMS/get/relval/13711/38.html. N.B.: the plots will be missing also in `CMSSW_11_0_0_pre5` 
If you agree I can add it in this PR even if its orginal purpose was different.

Reply:
@mmusich please, add CTPPS also to the FakeHLT sequence. This would restore completely the validation setup of CTPPS. Thank you very much!

Reply:
done in the last commit. I tested it locally to behave. I think it should be ready for a new round of tests now.

Reply:
Thanks @mmusich for adding CTPPS!